### DISCUSSION TITLE
Simplify Admin API

### DISCUSSION THREAD

**Discussion by mkurapov** [interledger/rafiki#1823] [Ideas]
Instead of ASEs needing to store Rafiki-generated primary keys (eg `paymentPointerId` `assetId` ) etc, we can simplify the Admin API (mutations, queries) in a few instances to use (eg `paymentPointerUrl`, `assetCode`). This way, there is no need to store additional data from the ASE's perspective (for mutations that couldn't necessarily use the Admin UI)

**Comment by raducristianpopa** [comment]
I'm not entirely sure about using the asset code. If I recall correctly, we can have multiple [ISO4127 currency codes](https://www.iso.org/iso-4217-currency-codes.html) but with different scales? 🤔

**Reply by mkurapov** [reply]
Yes, but I don't know of a real life case (that we've encountered just yet) that would have the same asset code but a different asset scale. 

Additionally, it's something we could just enforce on asset creation or lookup: either assetCode is a unique key in the DB _or_ instead of assetId in mutations/queries, we can have required assetCode and optional assetScale. That way using assetCode just works, and if you want to support same assetCode but multiple scales you can just give it assetScale as well.

**Reply by raducristianpopa** [reply]
I actually like the idea of using the asset's code as a primary key. Or maybe a composite key with the asset's code and scale? I know for a fact that we already store `assetId` for each account that is being created in Testnet. I'm going to look into this on Testnet's side to see how it would affect it, if we opt to use the code as the primary key.
